block/mempool.go

```diff
@@ -86,11 +86,11 @@ func (m *LanedMempool) CountTx() int {
 }
 
 // GetTxDistribution returns the number of transactions in each lane.
-func (m *LanedMempool) GetTxDistribution() map[string]int {
-	counts := make(map[string]int, len(m.registry))
+func (m *LanedMempool) GetTxDistribution() map[string]uint64 {
+	counts := make(map[string]uint64, len(m.registry))
 
 	for _, lane := range m.registry {
-		counts[lane.Name()] = lane.CountTx()
+		counts[lane.Name()] = uint64(lane.CountTx())
```

Comment:
What type does `CountTx()` return? should this also be a `uint64`?

Reply:
the mempool interface is exported from upstream sdk, which unfortunately exports int instead of uints.